src/mdtest.c

```diff
@@ -76,6 +76,10 @@
 
 #include <mpi.h>
 
+#ifdef HAVE_LUSTRE_LUSTREAPI_H
```

Comment:
This will break if `linux/lustre/lustre_user.h` is the path to `lustre_user.h` (see #189)

Reply:
Please see note above.